lib/configGenerator/generateTerraformConfig.js

```diff
@@ -6,6 +6,7 @@ async function generateTerraformConfig(network, networkName, masternodesCount) {
     logs_count: 1,
     masternode_count: masternodesCount,
     main_domain: '""',
+    metrics_enabled: false,
```

Comment:
I added this so the generated `.tfvars` file provides an easy reminder of what setting needs to be changed to enable the metrics services. This PR still needs testing when set to false though, I tried setting up a devnet with no knowledge of this variable existing and got the following errors:
```

│ Error: Invalid index
│ 
│   on main.tf line 161, in resource "aws_route53_record" "metrics":
│  161:   records = [aws_instance.metrics[count.index].public_ip]
│     ├────────────────
│     │ aws_instance.metrics is empty tuple
│     │ count.index is 0
│ 
│ The given key does not identify an element in this collection value: the collection has no elements.
╵
╷
│ Error: Invalid index
│ 
│   on security_groups.tf line 344, in resource "aws_security_group" "masternode":
│  344:       "${aws_eip.metrics[0].public_ip}/32",
│     ├────────────────
│     │ aws_eip.metrics is empty tuple
│ 
│ The given key does not identify an element in this collection value: the collection has no elements.
```

It should still be possible to set up a network without Grafana without encountering errors.

Reply:
Oooh that's interesting, I did test this but with the recent change to EIP it seems to be failing. I did change to EIP in order to connect to an MN though, if we don't need to do that and can set up mn-evo-services on a non-MN then it may not be needed anymore anyway.

I'll play around and push something later